### ISSUE TITLE
Twitch Stream Announcements keep Announcing Offline Streams

### ISSUE THREAD

**Issue by NeddyPlays** [Kwoth/NadekoBot#2268]
Hi,

I have been hosting the NadekoBot myself and setup Twitch Stream Announcements. Since the last version there is .streamoff switch that iam using. It says it does not announce offline streams anymore, but it still does. I've tried switching it on and off again and also did reboot the bot.

**Comment by zfbx** [comment]
This doesn't seem related... I am getting that error also but not at the same time as what this is doing. this seems more related to a potential typo in code for handling multiple of the same stream I would think. It works for the twitch channels only used once but the repeated one bugs it out

**Comment by xnaas** [comment]
Ah, alright. Interesting.

**Comment by Kwoth** [comment]
@zfbx remove all streams, and add it once and set a stream message and see if it happens again. The original issue has nothing to do with yours, so if you have an problem, create a new issue with reproducible steps.

**Comment by NeddyPlays** [comment]
@zfbx yes exactly the same! I get the stream offline message at another time then the other error messages in the logs

**Comment by NeddyPlays** [comment]
Still does it :(

At 16:16 a Streamer went offline, at that time i only got 
0 16:16:30 StreamNotificationService | Retreived and published stream statuses in 0.22s

In the console. Even when i got the Offline Announcements off.
